java.lang.NoSuchMethodException: com.microsoft.playwright.Frame$FillOptions.<init>()

Hi!

I'm having trouble deploying a quarkus app with quarkus-playwright used for runtime scrapping, everything works well locally but I encounter the following exception when running inside a docker image : 

```
com.microsoft.playwright.PlaywrightException: Internal error
	at com.microsoft.playwright.impl.Utils.convertType(Utils.java:83)
	at com.microsoft.playwright.impl.LocatorImpl.fill(LocatorImpl.java:250)
	at com.microsoft.playwright.Locator.fill(Locator.java:3072)
	at me.ldclrcq.filature.sources.connectors.free.FreeConnector.downloadDocuments(FreeConnector.java:54)
	at me.ldclrcq.filature.sources.connectors.free.FreeConnector_ClientProxy.downloadDocuments(Unknown Source)
	at me.ldclrcq.filature.synchronizations.Synchronizer.synchronize(Synchronizer.java:55)
	at java.base@21.0.7/java.util.Optional.ifPresent(Optional.java:178)
	at me.ldclrcq.filature.synchronizations.Synchronizer.synchronizeNow(Synchronizer.java:43)
	at me.ldclrcq.filature.synchronizations.Synchronizer_ClientProxy.synchronizeNow(Unknown Source)
	at me.ldclrcq.filature.connections.ConnectionResource.synchronizeNow(ConnectionResource.java:59)
	at me.ldclrcq.filature.connections.ConnectionResource_Subclass.synchronizeNow$$superforward(Unknown Source)
	at me.ldclrcq.filature.connections.ConnectionResource_Subclass$$function$$5.apply(Unknown Source)
	at io.quarkus.arc.impl.AroundInvokeInvocationContext.proceed(AroundInvokeInvocationContext.java:73)
	at io.quarkus.arc.impl.AroundInvokeInvocationContext.proceed(AroundInvokeInvocationContext.java:62)
	at io.quarkus.narayana.jta.runtime.interceptor.TransactionalInterceptorBase.invokeInOurTx(TransactionalInterceptorBase.java:136)
	at io.quarkus.narayana.jta.runtime.interceptor.TransactionalInterceptorBase.invokeInOurTx(TransactionalInterceptorBase.java:107)
	at io.quarkus.narayana.jta.runtime.interceptor.TransactionalInterceptorRequired.doIntercept(TransactionalInterceptorRequired.java:38)
	at io.quarkus.narayana.jta.runtime.interceptor.TransactionalInterceptorBase.intercept(TransactionalInterceptorBase.java:61)
	at io.quarkus.narayana.jta.runtime.interceptor.TransactionalInterceptorRequired.intercept(TransactionalInterceptorRequired.java:32)
	at io.quarkus.narayana.jta.runtime.interceptor.TransactionalInterceptorRequired_Bean.intercept(Unknown Source)
	at io.quarkus.arc.impl.InterceptorInvocation.invoke(InterceptorInvocation.java:42)
	at io.quarkus.arc.impl.AroundInvokeInvocationContext.perform(AroundInvokeInvocationContext.java:30)
	at io.quarkus.arc.impl.InvocationContexts.performAroundInvoke(InvocationContexts.java:27)
	at me.ldclrcq.filature.connections.ConnectionResource_Subclass.synchronizeNow(Unknown Source)
	at me.ldclrcq.filature.connections.ConnectionResource$quarkusrestinvoker$synchronizeNow_b369757ad6e6f8fca15824671793fd59f7b26a2f.invoke(Unknown Source)
	at org.jboss.resteasy.reactive.server.handlers.InvocationHandler.handle(InvocationHandler.java:29)
	at io.quarkus.resteasy.reactive.server.runtime.QuarkusResteasyReactiveRequestContext.invokeHandler(QuarkusResteasyReactiveRequestContext.java:141)
	at org.jboss.resteasy.reactive.common.core.AbstractResteasyReactiveContext.run(AbstractResteasyReactiveContext.java:147)
	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$15.runWith(VertxCoreRecorder.java:637)
	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2651)
	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2630)
	at org.jboss.threads.EnhancedQueueExecutor.runThreadBody(EnhancedQueueExecutor.java:1622)
	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1589)
	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base@21.0.7/java.lang.Thread.runWith(Thread.java:1596)
	at java.base@21.0.7/java.lang.Thread.run(Thread.java:1583)
	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
Caused by: java.lang.NoSuchMethodException: com.microsoft.playwright.Frame$FillOptions.<init>()
	at java.base@21.0.7/java.lang.Class.checkMethod(DynamicHub.java:1078)
	at java.base@21.0.7/java.lang.Class.getConstructor0(DynamicHub.java:1241)
	at java.base@21.0.7/java.lang.Class.getDeclaredConstructor(DynamicHub.java:2930)
	at com.microsoft.playwright.impl.Utils.convertType(Utils.java:62)
	... 39 more
```

My dockerfile look like this : 

```dockerfile
FROM mcr.microsoft.com/playwright:v1.53.0

WORKDIR /work/

RUN chown 1001:root /work \
    && chmod g+rwX /work \
    && chown 1001:root /work

COPY --chown=1001:root target/*.properties target/*.so /work/
COPY --chown=1001:root target/*-runner /work/application

RUN chmod ugo+x /work/application

EXPOSE 8080
USER 1001

CMD ["./application", "-Dquarkus.http.host=0.0.0.0"]
```
I suspected an issue with the library's playwright version not matching the docker image, but I have tried all recent versions without success (1.48, 1.50, 1.51, 1.52, 1.53).

--- a/deployment/src/main/java/io/quarkiverse/playwright/deployment/PlaywrightProcessor.java
+++ b/deployment/src/main/java/io/quarkiverse/playwright/deployment/PlaywrightProcessor.java
@@ -12,6 +12,7 @@ import org.jboss.jandex.DotName;
 
 import com.microsoft.playwright.Browser;
 import com.microsoft.playwright.ElementHandle;
+import com.microsoft.playwright.Frame;
 import com.microsoft.playwright.Playwright;
 import com.microsoft.playwright.impl.driver.jar.DriverJar;
 import com.microsoft.playwright.options.HttpHeader;
@@ -77,6 +78,7 @@ class PlaywrightProcessor {
         classNames.add(ElementHandle.ScreenshotOptions.class.getName());
         classNames.add(ElementHandle.ScrollIntoViewIfNeededOptions.class.getName());
         classNames.add(ElementHandle.SelectTextOptions.class.getName());
+        classNames.add(ElementHandle.SetCheckedOptions.class.getName());
         classNames.add(ElementHandle.SetInputFilesOptions.class.getName());
         classNames.add(ElementHandle.TapOptions.class.getName());
         classNames.add(ElementHandle.TypeOptions.class.getName());
@@ -86,6 +88,36 @@ class PlaywrightProcessor {
         classNames.add(HttpHeader.class.getName());
         classNames.add(Timing.class.getName());
         classNames.add(ViewportSize.class.getName());
+        classNames.add(Frame.AddScriptTagOptions.class.getName());
+        classNames.add(Frame.AddStyleTagOptions.class.getName());
+        classNames.add(Frame.CheckOptions.class.getName());
+        classNames.add(Frame.ClickOptions.class.getName());
+        classNames.add(Frame.DblclickOptions.class.getName());
+        classNames.add(Frame.DispatchEventOptions.class.getName());
+        classNames.add(Frame.FillOptions.class.getName());
+        classNames.add(Frame.FocusOptions.class.getName());
+        classNames.add(Frame.GetAttributeOptions.class.getName());
+        classNames.add(Frame.GetByRoleOptions.class.getName());
+        classNames.add(Frame.GetByTextOptions.class.getName());
+        classNames.add(Frame.HoverOptions.class.getName());
+        classNames.add(Frame.InnerHTMLOptions.class.getName());
+        classNames.add(Frame.InnerTextOptions.class.getName());
+        classNames.add(Frame.InputValueOptions.class.getName());
+        classNames.add(Frame.IsVisibleOptions.class.getName());
+        classNames.add(Frame.LocatorOptions.class.getName());
+        classNames.add(Frame.PressOptions.class.getName());
+        classNames.add(Frame.SelectOptionOptions.class.getName());
+        classNames.add(Frame.SetContentOptions.class.getName());
+        classNames.add(Frame.SetInputFilesOptions.class.getName());
+        classNames.add(Frame.TapOptions.class.getName());
+        classNames.add(Frame.TextContentOptions.class.getName());
+        classNames.add(Frame.TypeOptions.class.getName());
+        classNames.add(Frame.UncheckOptions.class.getName());
+        classNames.add(Frame.UncheckOptions.class.getName());
+        classNames.add(Frame.WaitForFunctionOptions.class.getName());
+        classNames.add(Frame.WaitForLoadStateOptions.class.getName());
+        classNames.add(Frame.WaitForSelectorOptions.class.getName());
+        classNames.add(Frame.WaitForURLOptions.class.getName());
         classNames.addAll(collectImplementors(combinedIndex, Playwright.class.getName()));
 
         //@formatter:on

